tools/Config.mk

```
@@ -364,7 +364,7 @@ endef
 
 define ASSEMBLE
 	@echo "AS: $1"
-	$(Q) $(CCACHE) $(CC) -c $(AFLAGS) $1 $($(strip $1)_AFLAGS) -o $2
+	$(Q) $(CCACHE) $(CC) -c $(AFLAGS) $(abspath $1) $($(strip $1)_AFLAGS) -o $(abspath $2)
```

please create a new patch to algin with other:
```suggestion
	$(Q) $(CCACHE) $(CC) -c $(AFLAGS) $($(strip $1)_AFLAGS) $(abspath $1) -o $(abspath $2)
```